packages/slice-machine/src/legacy/lib/builders/SliceBuilder/FieldZones/index.tsx

```diff
@@ -212,12 +205,10 @@ const FieldZones: FC = () => {
         }
         testId="static-zone-content"
         isRepeatableCustomType={undefined}
-        emptyStateHeading={
-          groupsInSlicesExperiment.eligible ? undefined : "No fields"
-        }
+        emptyStateHeading={undefined}
```

Comment:
Is there a particular reason that we specify `undefined` for all the unused props here?

Reply:
That's a very good question. I have no idea. These are ollllddd components written in JSX not TSX. 

Maybe it's done like this because since it's not typed, you don't know the props at all. Because of this way, you know exactly what you are passing or not. 
I preferred keeping that way since I like consistency, but we could change all that of course.